<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <widget class="StageTest" name="vTest"/>
   <widget class="StageSetting" name="vSetting"/>
   <widget class="StageBarcode" name="vBarcode"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StageSetting</class>
   <extends>QWidget</extends>
   <header>StageSetting.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StageBarcode</class>
   <extends>QWidget</extends>
   <header>StageBarcode.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StageTest</class>
   <extends>QWidget</extends>
   <header>StageTest.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
